robottelo/test.py

```diff
@@ -238,11 +238,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        dir_structure = '{screenshots_dir}/{date}/{dir}'.format(
```

Comment:
Can we use use `os.path.join` to create this path name? This is fragile. For example, notice how `screenshots_dir` is `/tmp/robottelo/screenshots/` in the config file, which will lead to this output:

    /tmp/robottelo/screenshots//{date}/{dir}